prow/gcsupload/run.go

```diff
@@ -17,18 +17,15 @@ limitations under the License.
 package gcsupload
 
 import (
-	"context"
 	"fmt"
 	"mime"
+	"net/url"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
 
-	"cloud.google.com/go/storage"
 	"github.com/sirupsen/logrus"
```

Comment:
Please keep our imports in their own group

Reply:
Ups sorry. Of course I'll check all files to be sure. Maybe should fork goimports for that feature :P

Reply:
Okay wasn't aware of the -local option of goimports. Hope this fixes it for good..